src/status_im/contexts/wallet/home/tabs/assets/view.cljs

```diff
@@ -15,6 +15,6 @@
         :parent-height 560
         :animated?     false}]
       [rn/flat-list
-       {:render-fn               quo/token-value
+       {:render-fn               (fn [item & _rest] [quo/token-value item])
```

Comment:
@vkjr, could you expand on why are we wrapping the function `token-value`?

Reply:
Since we have a recommendation to only pass non-anonymous functions to `flat-list`, I think it would be better to extract to a separate function. It's verbose, but at least we don't pay any cost in production runtime.

```diff
modified   src/status_im/contexts/wallet/home/tabs/assets/view.cljs
@@ -5,6 +5,10 @@
     [status-im.contexts.wallet.home.tabs.assets.style :as style]
     [utils.re-frame :as rf]))
 
+(defn- token-value
+  [item & _]
+  [quo/token-value item])
+
 (defn view
   []
   (let [tokens-loading?  (rf/sub [:wallet/tokens-loading?])
@@ -15,6 +19,6 @@
         :parent-height 560
         :animated?     false}]
       [rn/flat-list
-       {:render-fn               (fn [item & _rest] [quo/token-value item])
+       {:render-fn               token-value
         :data                    tokens
         :content-container-style style/list-container}])))
```